terraform/setup/vpc.tf

```diff
@@ -10,9 +10,25 @@ resource "aws_security_group" "ec2_security_group" {
   vpc_id = data.aws_vpc.default.id
 
   egress {
-    from_port   = 0
-    to_port     = 0
-    protocol    = "-1"
+    from_port   = 443
```

Comment:
what is the purpose of these port mappings?

Reply:
443 is for HTTPS. So we're restrict outbound access to HTTPS, except for HTTP for communicating with IMDS, and NFS for the EFS mounting.